### PR TITLE
docs: clarify skipping running ad tests

### PR DESCRIPTION
**Pull request by justin-stephenson**
Related to https://github.com/next-actions/pytest-mh/issues/9

### CODE REVIEW

**Review by ikerexxe — DISMISSED** [review]
LGTM. Thanks for the patch!

File: docs/running-tests.rst [hunk 1]
```diff
@@ -151,7 +151,7 @@ installation.
 .. note::
 
     It is not required to have the Active Directory machine running in order to
-    run the tests. If you run the tests with ``--mh-lazy-ssh`` (as shown in the
+    run the tests. If you run the tests with ``--mh-lazy-ssh -k "not ad"`` (as shown in the
```
Comment (pbrezina) [inline docs/running-tests.rst:154]: Can you please wrap it at 80 chars?

**Review by pbrezina — CHANGES_REQUESTED** [review]
Thank you, just a nitpick.

File: docs/running-tests.rst (hunk 1 above)
Comment (justin-stephenson) [reply docs/running-tests.rst:154]: Done.

Comment (pbrezina) [reply docs/running-tests.rst:154]: I think you forgot to push it :-)

Comment (justin-stephenson) [reply docs/running-tests.rst]: Fixed, this PR was not from my fork by accident.

**Review by ikerexxe — APPROVED** [review]

**Review by pbrezina — APPROVED** [review]

**Pull request merged**